modules/examiner/php/editexaminer.class.inc

```diff
@@ -270,11 +270,12 @@ class EditExaminer extends \NDB_Form
                 }
             }
         }
-
-        $baseurl = \NDB_Factory::singleton()->settings()->getBaseURL();
-        $url     = "{$baseurl}/examiner/editExaminer/?identifier="
-               . $examinerID;
+        $factory = \NDB_Factory::singleton();
+        $baseURL = $factory->settings()->getBaseURL();
+        $url     = $baseURL."/examiner/editExaminer/?identifier="
+                   . $examinerID;
         header("Location: " . $url, true, 303);
+        exit();
```

Comment:
This should not be hard exiting and the rest of the code seems to only be changing the case of the variable.. what are you trying to accomplish with this PR?

Reply:
header location don't work without exit() or die(). I am fixing the header function.